Rakefile

```diff
@@ -161,30 +177,55 @@ task :release_no_checks do
   puts "New version will be #{green(version)}"
 
   version_string = version.format(VERSION_FORMAT)
-  puts "Updating podspec."
-  contents = File.read(PODSPEC)
-  contents.gsub!(/s\.version\s*=\s(:?'|")\d+\.\d+\.\d+(-\w+\.\d)?(:?'|")/, "s.version          = \"#{version_string}\"")
-  File.open(PODSPEC, 'w') { |file| file.puts contents }
-  abort red "Podspec should have been updated with the new version, but it wasn't." unless file_is_dirty(PODSPEC)
 
+  task(:update_version_in_podspec).invoke(UIKIT_PODSPEC, version_string)
+  task(:update_version_in_podspec).reenable
+  task(:update_version_in_podspec).invoke(SWIFTUI_PODSPEC, version_string)
+  task(:check_changelog_version).invoke(version_string)
```

Comment:
only need to call `reenable` on tasks that will be called more than once. This is not the case so no need to call it everywhere :)